CONTRIBUTING.adoc

```diff
@@ -0,0 +1,74 @@
+= Contributing to the mobile cli
+
+This document explains how to set up a development environment and get involved with Mobile CLI project.
+
+Before anything else, https://help.github.com/articles/fork-a-repo/[fork] the Mobile CLI project.
+
+== Tools We use
+
+* Git
```

Comment:
would be helpful if each of these were a link to install docs for the requirement

Reply:
I dunno how helpful that would be. If you are actually going to start writing code links to install git and go don't seem that helpful?